superset-frontend/src/components/PopoverSection/PopoverSection.test.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import userEvent from '@testing-library/user-event';
+import PopoverSection from 'src/components/PopoverSection';
+
+test('renders with default props', () => {
+  render(
+    <PopoverSection title="Title">
```

Comment:
It probably doesn't make sense with these simpler components, but it may be worth considering in other instances: rather than repeating the scaffolding JSX here to assert against, you could import the story (i.e. `InteractivePopoverSection`) from the storybook file, and render _that_, so the tests and storybooks are a little more closely aligned.